avro-codegen/src/main/java/com/linkedin/avroutil1/codegen/SpecificRecordGeneratorUtil.java

```diff
@@ -93,8 +93,25 @@ public class SpecificRecordGeneratorUtil {
       CompatibleSpecificRecordBuilderBase.class.getName()
   ));
 
+  private static final ClassValue<Enum<?>[]> ENUM_CONSTANTS = new ClassValue<Enum<?>[]>() {
```

Comment:
putthing this here means generated classes will have a runtime dependency on the avro-codegen module. we'd like to keep helper as the only RUNTIME code dependency, so you should move this utility method (and the ClassValue backing it) to a util class in the helper module

Reply:
Moved to com.linkedin.avroutil1.Enums inside the helper module.